core/templates/dev/head/base.html

```diff
@@ -280,6 +281,16 @@
               {{ warnings_and_loader() }}
             </div>
 
+            <noscript>
+              <div class="oppia-page-cards-container">
+                <div class="md-default-theme oppia-page-card oppia-long-text">
+                  <h2>We Need JavaScript in Your Browser <img src="/images/general/sad_face.png" /> </h2>
```

Comment:
Hmm, my browser can't find the sad face, did you remember to add it commit? :) 

The direction of the wording is better, but perhaps something like: 
`Looks like JavaScript isn't enabled on your browser :( `
might be better.